bin/ember-template-lint.js

```diff
@@ -481,7 +482,6 @@ async function run() {
   let hasTodos = options.includeTodo && results.todoCount;
   let hasUpdatedTodos = options.updateTodo;
 
-  let Printer = require('../lib/formatters/default');
```

Comment:
This is now a static import at top of file. Could possibly be moved back to dynamic `await import()` but static import seems fine.

Reply:
I think this is fine since we're always loading a printer.

Reply:
Okay, I did this in a few places since static imports don't require switching stuff to await, but probably wouldn't be difficult to move it back to dynamic (using `await import()`) later if needed.